README.md

```diff
@@ -42,7 +42,7 @@ Installation steps:
     * run `rake db:migrate`
     * OR
     * in MySQL, `drop database XXX;` for each database in `config/database.yml` and then try `rake db:setup` again
-* `rake db:seed` to populate it with initial data
+* `rake db:seed` to populate it with initial dummy data
```

Comment:
is it dummy data? I thought it was just the most basic data required to get up and running. "dummy data" makes it sound like it'd be for running tests only, but not for bootstrapping a brand spanking new and potentially production-ready system.